backend/src/main/kotlin/io/zell/zdb/state/blacklist/BlacklistState.kt

```diff
@@ -16,7 +15,8 @@ class BlacklistState(private var readonlyDb: ReadonlyTransactionDb) {
 
         val processInstanceKey = DbLong()
         val blackListColumnFamily =
-            readonlyDb.createColumnFamily(ZbColumnFamilies.BLACKLIST,
+            readonlyDb.createColumnFamily(
+                ZbColumnFamilies.BLACKLIST,
```

Comment:
:question: Do you use google format? I think I need to configure the project settings in the repo, and reformat the code. I actually have this on my todo list :sweat_smile:

Reply:
I don't think I enabled google-java-format but still my IntelliJ had very different _opinions_ on formatting :sweat_smile:. I tried to keep formatting changes to a minimum though.